apps/dashboard/.env.example

```
@@ -8,16 +8,30 @@
+# "ws_local_root" for local
 UNKEY_WORKSPACE_ID=
+# "api_local_root_keys" for local
 UNKEY_API_ID=
 
+# 'workos' or 'local'
+AUTH_PROVIDER=
+
+# Set by docker. Use http://localhost:8080 for local
+AGENT_URL=
+# Set by docker. use "agent-auth-secret" for local
+AGENT_TOKEN=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The AGENT_TOKEN key should go before the AGENT_URL key

(UnorderedKey)